components/payments/settings/payments-webhooks-table.tsx

```diff
@@ -211,15 +235,56 @@ export const PaymentsWebhooksTable: React.FC<PaymentsWebhooksTableProps> = ({
     data.push({ url: "Enter Webhook Url" });
   }
 
-
   return (
-    <TWTable
-      title="webhooks"
-      data={data}
-      columns={columns}
-      isLoading={isLoading}
-      isFetched={isFetched}
-    />
+    <>
+      <Modal isOpen={isOpen} onClose={onClose} isCentered>
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader>Delete webhook</ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            {webhookToRevoke && (
```

Comment:
what if you... don't have a webhook to revoke? 

I don't think this is the way to go about this, can you check how I did the delete confirmation in engine and do something similarr? You can check the RemoveModal on `admins-table` you should have the ID before clicking the button so you can always render the modal knowing what you're deleting